server/graphql/v2/mutation/TierMutations.ts

```diff
@@ -152,6 +147,10 @@ const tierMutations = {
       checkRemoteUserCanUseAccount(req);
 
       const tier = await fetchTierWithReference(args.tier, { throwIfMissing: true });
+      if (tier === 'custom') {
+        throw new Error('Cannot delete custom tier. Set settings.disableCustomContributions to true instead.');
+      }
+
```

Comment:
Had to do this because TS (rightfully) complained that `req.loaders.Collective.byId.load('custom')` was not allowed.